subsys/bluetooth/shell/bt.c

```diff
@@ -241,6 +284,9 @@ static struct bt_conn_cb conn_callbacks = {
 #if defined(CONFIG_BT_SMP) || defined(CONFIG_BT_BREDR)
 	.security_changed = security_changed,
 #endif
+#if defined(CONFIG_BT_REMOTE_INFO)
+	.remote_info_available = remote_info_available
```

Comment:
Add a comma to the end of this line, in case the struct gets extended in the future

Reply:
Good point. 👍